src/DbHelper.php

```diff
@@ -87,6 +92,8 @@ public static function dropTable(ConnectionInterface $db, string $table = '{{%lo
         $command = $db->createCommand();
         $tableRawName = $db->getSchema()->getRawTableName($table);
 
+        self::validateSupportedDatabase($db);
```

Comment:
Liekly any new driver for SQL'99 compatible DB will work so I see no need for the check.